doc/changelog.rst

```diff
@@ -4,9 +4,42 @@ Version history and changelog
 =============================
 
 
-Version 1.0.0-alpha (2018)
+Version 1.0.0 (2020-06-30)
 --------------------------
 
+Code
+~~~~
+
+- Now requires Fortran 2008
+- Rewrote linked list functionality for caching to instead use (reallocating) arrays
+- "Number of components" marker in rsp_tensor output file now written as 'NUM_COMPONENTS' instead of 'NUM COMPONENTS'
+- Significantly decreased usage of array constructors in function/subroutine arguments
+- Now compiles and runs with most compilers but still problems with some Intel/2018 and Intel/2019 setups
+- Fixed various memory leak/out-of-bounds errors that sometimes would happen
+- OpenRSP now looks for available file units before choosing one to use
+- Disabled internal memory limit and memory bookkeeping, may be reinstated later
+- If a response tensor is large, then if it's printed at the debugging print level, it's broken down into smaller chunks
+- Added stops for various currently unsupported residue calculation setups
+- Fixed a bug concerning testing of perturbation frequencies against excitation energy for residue calculations
+- Removed some unused residue-related routines
+- Calculation setup errors encountered in wrapper routines now cause exit, not just warning and then continuing
+- The "excitation" perturbation in a single residue calculation is now given
+  the label EX1 in the rsp_tensor file; however, its current implementation
+  still results in triplication of the calculation result data due to being
+  treated as having three components when in fact it has got only one
+
```

Comment:
I would add things mentioned as minor issues here to a list of "Known issues" in the release description here on GitHub.

Reply:
OK, adjusting this.